rover_msgs/ThermistorData.lcm

```diff
@@ -1,5 +1,6 @@
 package rover_msgs;
 
 struct ThermistorData {
-	double temperature; //Celsius
-}
+	double temperature_C; // Celsius
```

Comment:
There are some weird changes in the science LCM messages but this can be fixed when we merge and pr ish-integration

Reply:
interesting again why science bridge changes were here too...

Reply:
I thought I got all of the random GUI changes out of the IK changes last summer so I'm confused also, but agree that it theoretically should be fine since none of the GUI code in main really is relevant anyways